src/styles/card.scss

```
@@ -0,0 +1,112 @@
+.page__catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  grid-column-gap: 50px;
+  grid-row-gap: 48px;
+  padding: 51px 0;
+}
+
+.container {
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &__card {
+    height: 134px;
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px;
+
+  }
+
+  &__text {
+    display: flex;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 40px 0 0;
+    color: #060b35;
+
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 4px 0 0;
+    font-style: normal;
+    color: #616070;
+  }
+
+  &__star {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__review-text {
+    text-align: right;
+    font-style: normal;
+    margin: 0;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__key {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    color: #616070;
+  }
+
+  &__value {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-style: normal;
+    width: 166px;
+    height: 40px;
+    margin-top: 16px;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    transition: 0.4s;
```

use transition only for `background-color`
```suggestion
    transition: background-color 0.4s;
```